OOP/News.py

```diff
@@ -0,0 +1,90 @@
+# Создайте инструмент, который будет создавать новостную ленту, сгенерированную пользователем:
+# 1. Пользователь выбирает, какой тип данных он хочет добавить
+# 2. Предоставьте требуемые данные типа записи
+# 3.Запись публикуется в текстовом файле в специальном формате
+
+# Вам необходимо реализовать:
+# 1.Новости – текст и город в качестве ввода. Дата рассчитывается во время публикации.
+# 2.Приватное объявление – на входе текст и дата истечения срока действия. Оставшийся день рассчитывается во время публикации.
+# 3. Ваш уникальный с уникальными правилами публикации.
+
+# Created a constractor witch will add new publication to the txt file
+class File_creator:
+
+    def record_to_file(self, text):
+        with open("OOP/result.txt", "a+") as f:
+            f.write(text)
+
+# create a class wich will be creating a news
+class News(File_creator):
+
+    def __init__(self, city, text):
+        self.city = city
+        self.text = text
+    # created constructer wich will be creating a news and add it to the txt file
+
+    def new_constr(self):
+        import datetime
```

Comment:
I already mentioned before that all imports are in the very beginning of a file.

Reply:
fixed